src/components/toast/toast.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const Toast = (props) => (
+  <div>
+    <ToastContainer {...props} />
```

Comment:
Why do you set this component in `components `folder if you name it `ToastContainer`?

Reply:
it's removed